src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -0,0 +1,339 @@
+          </SidebarBlock>
+        </SidebarColumn>
+      </ColumnGrid>
+    );
+  }
+
+  renderError(error) {
+    const notFound = Object.values(this.state.errors).find(
+      resp => resp && resp.status === 404
+    );
+
+    if (notFound) {
+      return this.renderWrapper(<NotFound />);
+    }
+
+    return this.renderWrapper(super.renderError(error, true, true));
+  }
+
+  renderLoading() {
+    return this.renderWrapper(super.renderLoading());
+  }
+
+  renderWrapper(children: React.ReactNode) {
+    const {organization, location} = this.props;
+    const eventView = this.getEventView();
+    const {event} = this.state;
+
+    return (
+      <EventDetailsWrapper
+        organization={organization}
+        location={location}
+        eventView={eventView}
+        event={event}
+      >
+        {children}
+      </EventDetailsWrapper>
+    );
+  }
+}
+
+type EventDetailsWrapperProps = {
+  organization: Organization;
+  location: Location;
+  eventView: EventView;
+  event: Event | undefined;
+  children: React.ReactNode;
+};
+
+class EventDetailsWrapper extends React.Component<EventDetailsWrapperProps> {
+  getDocumentTitle = (): string => {
+    const {event, eventView} = this.props;
+
+    const titles = [t('Discover')];
+
+    const eventViewName = eventView.name;
+    if (typeof eventViewName === 'string' && String(eventViewName).trim().length > 0) {
+      titles.push(String(eventViewName).trim());
+    }
+
+    const eventTitle = event ? getTitle(event).title : undefined;
+
+    if (eventTitle) {
+      titles.push(eventTitle);
+    }
+
+    return titles.join(' - ');
+  };
+
+  render() {
+    const {organization, location, eventView, event, children} = this.props;
+
+    return (
+      <DocumentTitle title={`${this.getDocumentTitle()} - ${organization.slug} - Sentry`}>
+        <React.Fragment>
+          <PageHeader>
+            <DiscoverBreadcrumb
+              eventView={eventView}
+              event={event}
+              organization={organization}
+              location={location}
+            />
+          </PageHeader>
+          {children}
+        </React.Fragment>
+      </DocumentTitle>
+    );
+  }
+}
+
+const EventHeader = props => {
+  const {title} = getTitle(props.event);
+  return (
+    <div data-test-id="event-header">
+      <OverflowHeader>{title}</OverflowHeader>
+      <p>{getMessage(props.event)}</p>
+    </div>
+  );
+};
+EventHeader.propTypes = {
+  event: SentryTypes.Event.isRequired,
+};
+
+const OverflowHeader = styled('h2')`
+  line-height: 1.2;
+  ${overflowEllipsis}
+`;
+
+const MetadataContainer = styled('div')`
+  display: flex;
+  justify-content: space-between;
+
+  color: ${p => p.theme.gray3};
+  font-size: ${p => p.theme.fontSizeMedium};
+`;
+
+/**
+ * Render metadata about the event and provide a link to the JSON blob
+ */
+const EventMetadata = (props: {event: Event; eventJsonUrl: string}) => {
+  const {event, eventJsonUrl} = props;
```

Comment:
If you passed in the organization you could generate the JSON url here instead of in the parent.

Reply:
fixed in https://github.com/getsentry/sentry/pull/15562/commits/1583475a79d21f1031060bfa72e03010cb83e8b0